packages/slice-machine/lib/builders/SliceBuilder/index.tsx

```diff
@@ -146,4 +181,28 @@ const SliceBuilder: React.FunctionComponent = () => {
   );
 };
 
-export default SliceBuilder;
+const SliceBuilderWithRouter = () => {
+  const router = useRouter();
```

Comment:
defiantly pass the next router to connected router to keep them in sync :)

Reply:
changed!

Reply:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/69144671/177345669-c3f4513b-30f0-489d-98a1-9e6a382d0eff.png">
all the information about the location